includes/Modules/AdSense.php

```diff
@@ -310,44 +279,6 @@ protected function create_data_request( Data_Request $data ) {
 				}
 
 				return $this->create_adsense_earning_data_request( array_filter( $args ) );
-			case 'GET:notifications':
-				return function() {
-					$alerts = $this->get_data( 'alerts' );
```

Comment:
AdSense uses `getAlerts` directly which uses `GET:alerts` so this is no longer used.

Reply:
@aaemnnosttv This must not be removed. `getAlerts` is used during setup, but we still have legacy notifications functionality relying on this. The overall `GET:modules/adsense/notifications` endpoint will look if this particular datapoint here exists and use it, so this block needs to be kept in place.

Reply:
@felixarntz I know it's connected to that endpoint but I couldn't find where it's actually used. Can you point me to it?

Reply:
@aaemnnosttv It's used in the `Alert` component which is included on every module page for the respective module. Basically if some alert comes up in your AdSense account, you'll see it on the AdSense module page.

Reply:
Ah, I see it now. Apparently also used in `DashboardModulesAlerts`. Thanks!

Reply:
Restored 👍